### PR TITLE
Add explaining text when creating new profile

### LINKED ISSUE #29: More intuitive renaming of color profiles
It's hard to recreate a color profile under a different name. There should be an option to rename them.

### PR DESCRIPTION
**Pull request by ghost**
Closes #29

### CODE REVIEW

**Comment by ghost** [comment]
@fsobolev Could you check this quickly? It seems too simple to be right.

File: src/preferences_window.py [hunk 1]
```diff
@@ -342,7 +342,7 @@ def create_colors_page(self):
         self.profile_new_button_add.connect('clicked', \
             self.create_color_profile)
         self.profile_new_box.append(self.profile_new_button_add)
-        self.profile_new_label = Gtk.Label.new('')
+        self.profile_new_label = Gtk.Label.new(_('The new profile will be a copy of the active profile.'))
```
Comment (fsobolev) [inline src/preferences_window.py:345]: The same change is required on line 626

Comment (ghost) [reply src/preferences_window.py:345]: Ah, yes, I thought I was missing something.

I can fix it in a couple of hours, or you can if you want.

**Comment by fsobolev** [comment]
Thank you! 😊

**Pull request merged**